ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -143,6 +144,41 @@ objects:
           oc create -f /tmp/cluster/insights-live.yaml || true
         fi
 
+        # create fips enable helpers
```

Comment:
I would prefer this be a custom test (update TEST_COMMAND with your script) rather than have this be in the template for now.

Reply:
Basically you can have multiline bash in TEST_COMMAND, and it is appropriate for now that only this test enable FIPS like this.  Then you don't have to change the template.

Reply:
I asked Alex to change the template so we don't have that copied N times for every suite ... it's the same as the multitenant config. We'll run with this until we have a better mechanism than templates for it.

Reply:
I don't want this in the template.  This is config specific to a specific test today.

Reply:
Until it's proven itself it doesn't belong in the template

Reply:
I would agree once this reaches stable we can move it into the template, under the guidance of whatever the config options for the template are.  I don't want to make this problem worse until we have a reason to.

Reply:
That's exactly how the multitenant support got added, we didn't enforce that then. We won't enforce it now. I want to get this implemented for the KR and revisit once we have a stable and reasonable solution for this class of issue.

Reply:
Multitenant was well scoped.  This is a bit hacky.  To go in to the template it needs to be moderately correct.  So the choice is to the submitter - move it to TEST_COMMANDS and let it continue to be isolated (and potentially a bit tacky) or it needs to be general and correct (see my comments).  I'd prefer that the submitter iterate on this in an isolation fashion, because it's not currently general, and by doing that can get some familiarity with how the current approach is going to fail and its flake rate independently.